src/test/java/com/google/api/codegen/testdata/php_main_library.baseline

```diff
@@ -742,6 +742,8 @@ class LibraryServiceApi
      *          The edition of the series
      *     @type bool $reviewCopy
      *          If the book is in a pre-publish state
+     *     @type string $seriesUuid
+     *          Uniquely identifies the series to the publishing house.
```

Comment:
@bjwatson This field is optional in all languages -- the config generator marks it as required, but the config that is passed into the tests has it as optional.